Source/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry.h

```diff
@@ -60,7 +60,7 @@ typedef struct BSG_KSCrash_SentryContext {
     bool reportWhenDebuggerIsAttached;
 
     /** If true, will trace threads and report binary images. */
```

Comment:
nit: The comment's not quite accurate w.r.t. the enum

Reply:
Updated comment.